docs/usage/self-hosted-configuration.md

```diff
@@ -128,7 +128,9 @@ e.g.
 
 ## composerIgnorePlatformReqs
 
-Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer package manager.s
+Set the platform requirements you want to exclude when using Composer. Each item will be added to the composer command with `--ignore-platform-req`. When set to an empty array, `--ignore-platform-reqs` will be added to the Composer command.
+
+Set to `null` to prevent usage of `--ignore-platform-reqs` and `--ignore-platform-req` in the Composer package manager.
```

Comment:
This line feels incomplete.

First we say: set this to null to prevent usage of ..., and it seems you also need to do something when invoking `composer` on the command line (`--ignore-platform-req`) Are we sure this is clear enough for Composer users?

Reply:
I believe Composer users should know about the platform requirements and how to ignore them, but I updated the text to be more explicit.